### ISSUE TITLE
The image is not linked to the one on docker.io and differs from it

### ISSUE THREAD

**Issue by bexelbie** [LalatenduMohanty/rpm-of-docker#1]
Are we sure we want to have an image that inspects differently than the stock image we claim it is?  This could lead to a lack of faith for users basing derived images off of the ones we are shipping.

**Comment by LalatenduMohanty** [comment]
@bexelbie  good point. We should look in to this if we go by this route of delivering docker images in the vagrant box. However at this point I dont know.
